Remotion/Web/UnitTests/Core/ContentSecurityPolicy/CspEnabledHtmlTextWriterTest.cs

```diff
@@ -126,7 +126,7 @@ public void AddAttribute_WithSupportedEventType_ScriptIsRegistered (string event
                       _pageStub.Object,
```

Comment:
please check that all AddAtribute methods are excercised both for javascript and non-javascript.